.github/workflows/chromatic.yml

```
@@ -0,0 +1,23 @@
+name: "Chromatic"
```

Consider a more descriptive name 
```suggestion
name: "Visual Regression Tests - Chromatic"
```

---

I agree that this should be a descriptive name. I just went with the simplest name to get started 🙂